ci-operator/SECRETS.md

```diff
@@ -55,6 +55,15 @@ secrets currently exist:
 | `metrics-int.key`             | Azure Geneva metrics authentication key |
 | `system-docker-config.json`   | Root/node/system level docker config.json file, currently holding access registry.redhat.io |
 
+#### `cluster-secrets-openstack`
+
+|        Key        | Description |
+| ----------------- | ----------- |
+| `credentials`     | TODO |
+| `clouds.yaml`     | TODO |
+| `ssh-publickey`   | TODO |
+| `pull-secret`     | Credentials for pulling OpenShift images from Quay. |
```

Comment:
@flaper87: please fill in this section (either in a comment here and I'll include it or in a separate PR).

Reply:
`clouds.yaml` -> credentials for the openstack cloud. See `https://docs.openstack.org/python-openstackclient/pike/configuration/index.html`

`ssh-privatekey` - > Private half of the SSH key, for connecting to OpenStack Nova VMs.
`ssh-publickey` -> Public half of the SSH key, for connecting to OpenStack Nova VMs.


TBH, I've no idea what `credentials` is for. It's unclear to me if we're still using it but I believe it's got the same value as the `clouds.yaml` key. I think we can skip it for now